ground/src/main/java/com/google/android/ground/ui/surveyselector/SurveySelectorViewModel.kt

```diff
@@ -49,26 +49,22 @@ internal constructor(
   private val userRepository: UserRepository
 ) : AbstractViewModel() {
 
+  // TODO: Expose non-mutable state flow.
```

Comment:
Please add bug nos to new TODOs here and elsewhere.

Reply:
Done